### PR TITLE
configurable memory for cost_controller pods

### PR DESCRIPTION
**Pull request by myaser**
acting to a 24x7 alert

### CODE REVIEW

**Comment by RomanZavodskikh** [comment]
:+1:

**Comment by myaser** [comment]
👍🏻

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -1096,3 +1096,6 @@ sysctl_settings: ""
 
 # enables/disables the minDomains field for pod topology spread.
 min_domains_in_pod_topology_spread_enabled: "true"
+
+
+pod_deletion_cost_controller_memory: 200Mi
```
Comment (AlexanderYastrebov) [inline cluster/config-defaults.yaml:1101]: Make sense to prefix with `skipper_` (e.g. cluster registry policy may have a rule by prefix)

Comment (AlexanderYastrebov) [reply cluster/config-defaults.yaml:1101]: Was addressed by https://github.com/zalando-incubator/kubernetes-on-aws/pull/8001

**Pull request merged**